docs/faq/loading/Routine_load_faq.md

```diff
@@ -1,27 +1,81 @@
 
-**Issue description:**
+**Method 2:** **Increase the amount of data consumed by a Routine Load task from one or more partitions.**
 
-Failure to perform Routine Load to Kafka cluster. Errors are reported as:
+> **NOTICE**
+>
+> This method may cause delay in data loading.
 
-```plain text
-ErrorReason{errCode = 4, msg='Job failed to fetch all current partition with error [Failed to send proxy request: failed to send proxy request: [PAUSE: failed to create kafka consumer: No provider for SASL mechanism GSSAPI: recompile librdkafka with libsasl2 or openssl support. Current build options: PLAIN SASL_SCRAM]]'}
+The upper limit of the number of messages that a Routine Load task can consume is determined by either the parameter `max_routine_load_batch_size` which means the maximum number of messages that a load task can consume or the parameter `routine_load_task_consume_second` which means the maximum duration of message consumption. Once an load task consumes enough data that meets either requirement, the consumption is complete. These two parameters are FE dynamic parameters. For more information and the configuration method, see [Parameter configuration](../../administration/Configuration.md#loading-and-unloading).
+
+You can analyze which parameter determines the upper limit of the amount of data consumed by a load task by viewing the log in **be/log/be.INFO**. By increasing that parameter, you can increase the amount of data consumed by a load task.
+
+```Plaintext
+I0325 20:27:50.410579 15259 data_consumer_group.cpp:131] consumer group done: 41448fb1a0ca59ad-30e34dabfa7e47a0. consume time(ms)=3261, received rows=179190, received bytes=9855450, eos: 1, left_time: -261, left_bytes: 514432550, blocking get time(us): 3065086, blocking put time(us): 24855
 ```
 
-**Cause:**
+Normally, the field `left_bytes` in the log is greater than or equal to `0`, indicating that the amount of data consumed by a load task has not exceeded `max_routine_load_batch_size` within `routine_load_task_consume_second`. This means that a batch of scheduled load tasks can consume all data from Kafka without delay in consumption. In this scenario, you can set a larger value for `routine_load_task_consume_second` to increase the amount of data consumed by a load task from one or more partitions.
+
+If the field `left_bytes` is less than `0`, it means that the amount of data consumed by a load task has reached `max_routine_load_batch_size` within `routine_load_task_consume_second`. Every time data from Kafka fills the batch of scheduled load tasks. Therefore, it is highly likely that there is remaining data that has not been consumed in Kafka, causing delay in consumption. In this case, you can set a larger value for `max_routine_load_batch_size`.
+
+## What do I do if the result of SHOW ROUTINE LOAD shows that the load job is in the `PAUSED` state?
+
+- Check the field `ReasonOfStateChanged` and it reports the error message `Broker: Offset out of range`.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/795ff9a8-29d9-49ed-b8c7-ca20cf4d8492)
-> and see if it reports?